manifests/linux.pp

```diff
@@ -27,6 +27,10 @@
     ensure => $pkg_ensure,
   }
 
+  package { 'ebtables':
```

Comment:
Could you wrap this in a `$ebtables_manage` class parameter and default to false for users who don't need this package? Thanks!